lib/protocol/rest_json.js

```diff
@@ -75,7 +76,7 @@ function extractData(resp) {
     var body = resp.httpResponse.body;
     if (payloadMember.isEventStream) {
       parser = new JsonParser();
-      resp.data[payload] = util.createEventStream(
+      resp.data[rules.payload] = util.createEventStream(
```

Comment:
Looks like this was a miss in https://github.com/aws/aws-sdk-js/pull/2085, likely while doing copy-paste from `rest-xml.js`